src/solver/residuals.F90

```diff
@@ -28,7 +28,10 @@ subroutine residual_block
     integer(kind=intType) :: i, j, k, l
     integer(kind=intType) :: iale, jale, kale, lale, male ! For loops of ALE
     real(kind=realType), parameter :: K1 = 1.05_realType
-    real(kind=realType), parameter :: K2 = 0.6_realType ! Random given number
+
+    ! The line below is only used for the low-speed preconditioner part of this routine
+    real(kind=realType), parameter :: K2 = 639.048910156_realType ! t2_=0.5 for wsp=8
```

Comment:
Also, I think we can remove the changes to the low-speed preconditioner. This line especially doesn't make any sense to me.

Reply:
Yeah maybe @Aagaard87 can help here. I see how `K2` is used later on line 247 (but that variable `t2_` is not used anywhere, might be removed?) and on line 252 to calculate `betaMr2`.
There are a few comments that highlight doubts on the preconditioner implementation, and we know that it is not really working as intended.
@joanibal is planning to work on the low-speed preconditioner and might have some more insight.

Reply:
I agree with @sseraj . I think probably we can remove all changes to low speed preconditioner thing for now since we are not really using it.

Reply:
@SichengHe @sseraj I am cleaning this up, but leaving most of the comments because there might be value in them... are you ok with that?

Reply:
That's fine with me.